test/fixtures/dbt_project/.sqlfluff

```diff
@@ -5,3 +5,4 @@ dialect = postgres
 
 [sqlfluff:templater:dbt]
 profiles_dir = ../dbt
+project_dir = ./
```

Comment:
When running the existing tests we are now testing the functionality of linting while outside the project directory because we no longer change to the dbt project directory as part of the test fixture [in_dbt_project_dir()](https://github.com/sqlfluff/sqlfluff/pull/898/files#diff-02cf6c0e51d0bea24933771510432fa6d2b094d555c4ef5bfb53f65f075a1493L28). Instead we pass the fixture [project_dir()](in_dbt_project_dir) which is just the path of the dbt project and we change to that directory from outside as part on the new code int dbt.py.

Still, we're not testing the reading in the reading in of the project_dir from an .sqlfluff file so I'd be happy to add another test where we change directory to a directory with a .sqlfluff file and a subdirectory dbt_project.